Library/Homebrew/.rubocop.yml

```diff
@@ -41,6 +41,16 @@ Metrics/ModuleLength:
     - "dev-cmd/bottle.rb"
     - "test/**/*"
 
+Naming/InclusiveLanguage:
+  FlaggedTerms:
+    whitelist:
+      # For an option passed to `srb` in dev-cmd/typecheck.rb
+      AllowedRegex: "--error-white-list"
+    # TODO: If possible, make this stricter. Some instances may be unavoidable,
+    # such as sqlite_master in unpack_strategy/fossil.rb but these can be
+    # handled using AllowedRegex.
+    master:
```

Comment:
RuboCop added a new `Naming/InclusiveLanguage` cop in this release, which is enabled by default. Our codebase has many references to the `master` branch which is used as a fallback in many cases, or to refer to the `HEAD`/default branch of our official taps, which happen to be named `master`.

For the time being, I've replaced potentially problematic instances which can be safely replaced without breaking public API, but enabling the cop for the word "master" does not seem possible at this moment.

`brew style` gives me 39 offenses without disabling the check for "master", of which one seems to be a false positive (and I've mentioned this in the `TODO` comment).

Reply:
Makes sense, this is a nice middle-ground between reducing our use of the term but also allowing us to use this cop for other terms with our Git-heavy internal language.